test/test_unary_ufuncs.py

```diff
@@ -450,6 +450,64 @@ def test_out_arg_all_dtypes(self, device, dtype, op):
             else:
                 self._test_out_arg(op, input, out)
 
+    def _dtype_out_arg_helper(self, op, input, np_kwarg, torch_kwarg):
```

Comment:
Add a comment explaining what this function does and how it's intended to be used.